python/lvmdrp/functions/imageMethod.py

```diff
@@ -3217,6 +3235,13 @@ def detrend_frame(
         )
         detrended_img = detrended_img / numpy.ma.median(flat_array)
 
+    # add slitmap information if given
+    if in_slitmap is not None and detrended_img._header["IMAGETYP"] in {'flat', 'arc', 'object', 'science'}:
+        log.info("adding slitmap information")
+        detrended_img.setSlitmap(in_slitmap)
+    else:
+        log.warning("no slitmap information to be added")
```

Comment:
This will add the `slitmap` to basically every file that gets de-trended.  So every ancillary `lvm-cxxx` output file.   Is that what we want?  Do we need the slitmap in the arcs?  Just curious.

Reply:
Yes, that's what I want.